static/templates/settings/deactivation_stream_modal.hbs

```diff
@@ -1,13 +1,13 @@
 <div id="deactivation_stream_modal" class="modal modal-bg hide fade new-style" tabindex="-1" role="dialog" aria-labelledby="deactivation_stream_modal_label" aria-hidden="true">
     <div class="modal-header">
         <button type="button" class="close" data-dismiss="modal" aria-label="{{t 'Close' }}"><span aria-hidden="true">&times;</span></button>
-        <h3 id="deactivation_stream_modal_label">{{t "Delete stream" }} <span class="stream_name">{{stream_name}}</span></h3>
+        <h3 id="deactivation_stream_modal_label">{{t "Archive stream" }} <span class="stream_name">{{stream_name}}</span></h3>
     </div>
     <div class="modal-body">
-        <p>{{t "Deleting this stream will immediately unsubscribe everyone, and the stream's content will not be recoverable." }} <strong>{{t "Are you sure you want to do this?" }}</strong></p>
+        <p>{{t "Archiving this stream will immediately unsubscribe everyone and remove it from active streams." }} <strong>{{t "Are you sure you want to do this?" }}</strong></p>
```

Comment:
We probably will want to add a `?` link using `help_link_widget` adjacent to this explaining how this works at present; and then we'll plan to adjust the page that this links to as we work on this feature.

Reply:
Sounds good! The latest commit should address this. Please take a look when you get the chance and let me know how we should proceed. I will squash the commits when everything looks good!